### PR TITLE
Configurable polling interval for cluster widget

### PR DESCRIPTION
**Pull request by jsignell**
I did the scheduler_info for the client as well since it is a very similar situation.

Closes: #3665

### CODE REVIEW

File: distributed/distributed.yaml [hunk 1]
```diff
@@ -96,10 +96,12 @@ distributed:
     preload-argv: []        # See https://docs.dask.org/en/latest/setup/custom-startup.html
 
   client:
-    heartbeat: 5s  # time between client heartbeats
+    heartbeat: 5s  # Interval between client heartbeats
+    scheduler-info-update: 2s  # Interval between scheduler-info updates
```
Comment (jrbourbeau) [inline distributed/distributed.yaml:100]:
```suggestion
    scheduler-info-interval: 2s  # Interval between scheduler-info updates
```

File: distributed/distributed.yaml [hunk 2]
```diff
@@ -96,10 +96,12 @@ distributed:
...
 
   deploy:
     lost-worker-timeout: 15s  # Interval after which to hard-close a lost worker job
+    cluster-repr-update: 500ms  # Interval between calls to update cluster-repr for the widget
```
Comment (jrbourbeau) [inline distributed/distributed.yaml:104]:
```suggestion
    cluster-repr-interval: 500ms  # Interval between calls to update cluster-repr for the widget
```

File: distributed/client.py [hunk 3]
```diff
@@ -677,9 +677,13 @@ def __init__(
             heartbeat_interval = dask.config.get("distributed.client.heartbeat")
         heartbeat_interval = parse_timedelta(heartbeat_interval, default="ms")
 
+        scheduler_info_update = parse_timedelta(
+            dask.config.get("distributed.client.scheduler-info-update")
+        )
```
Comment (jrbourbeau) [inline distributed/client.py:682]: By default `parse_timedelta` will assume an input numeric value is in seconds, so the `* 1000` factor will be off in that case. I think we should specify `default="ms"` here, similar to when we parse `heartbeat_interval`

Comment (jsignell) [reply distributed/client.py:682]: Oh I see. I misunderstood what that default was for.

**Pull request merged**